docs/source/zero-code-change-limitations.rst

```diff
@@ -44,9 +44,71 @@ can also visually inspect the resulting cluster assignments.
 ``sklearn.decomposition.PCA``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The ``PCA`` implementation used by ``cuml.accel`` uses different SVD solvers
+than the ones in Scikit-Learn, which may result in numeric differences in the
+``components_`` and ``explained_variance_`` values. These differences should be
+small for ``svd_solver`` values of ``"auto"``, ``"full"``, or ``"arpack"``, but
+may be larger for randomized or less-numerically-stable solvers like
+``"randomized"`` or ``"covariance_eigh"``.
+
+Likewise, note that the implementation in ``cuml.accel`` currently may result
+in some of the vectors in ``components_`` having inverted signs. This result is
+not incorrect, but can make it harder to do direct numeric comparisons without
+first normalizing the signs. One common way of handling this is by normalizing
+the first non-zero values in each vector to be positive. You might find the
+following ``numpy`` function useful for this.
+
+.. code-block:: python
+
+    import numpy as np
+
+    def normalize(components):
```

Comment:
Ideally we'd fix our normalization to match sklearns, but for now I figured adding some code on how to do this would be useful. I went for a normalization function rather than one that also called `allclose` or something to avoid specifying acceptable tolerances. We should be "close" by some definition of close for anything except the randomized algorithms.